api/pagination/pagination.go

```diff
@@ -15,6 +15,9 @@ func StartAndEndPage(pageToken string, pageSize, totalSize int) (int, int, strin
 	if pageToken == "" {
 		pageToken = "0"
 	}
+	if pageSize < 0 || totalSize < 0 {
```

Comment:
I think we want to handle totalSize == 0 too. Shouldn't it be `<=`?

Reply:
I think total size 0 is fine just like pageSize == 0 🤔 at least how i interpreted we shouldn't error on total size = 0

Reply:
Yeah its the main reason I haven't changed it, because while negative values break how we structure api responses, 0 values will just lead us to return nothing.